wasm/jit/jit_amd64_test.go

```diff
@@ -2301,6 +2301,326 @@ func TestAmd64Compiler_compileSub(t *testing.T) {
 	})
 }
 
+func TestAmd64Compiler_compileMul(t *testing.T) {
+	t.Run("int32", func(t *testing.T) {
+		for _, tc := range []struct {
+			name         string
+			x1Reg, x2Reg int16
+		}{
+			{
+				name:  "x1:rax,x2:random_reg",
```

Comment:
if we like to use `rax` vs `reg_ax` let's make a constant file now then everything can have parity. We don't need to be beholdent to the imported names I think, but we should be consistent.

Reply:
`reg_ax` is a variable name only exiting in x86 assembler pkg, but the true name is `RAX`. So I will stack to RAX or whatever in doc or comments